bin/mujin_controllerclientpy_applyconfig.py

```diff
@@ -1,14 +1,35 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
+import os
 import six
 import copy
 import json
 import argparse
 import tempfile
 import subprocess
+import tarfile
+import io
+import cStringIO
 from mujincontrollerclient.controllerclientbase import ControllerClient
 
+try:
+    from mujincommon import EnsureDirectory
```

Comment:
@ziyan this is a circular dependency since mujincommon depends on mujincontrollerclientpy...